pages/index.vue

```diff
@@ -28,7 +29,7 @@
     const modQuery: QueryBuilderParams = { 
         path: '/modules', 
         where: [{ visibility: 'visible' }], 
-        sort: [{ id:1, $numeric:true }] 
+        sort: [{ order:1, $numeric:true }] 
```

Comment:
Hi @JaroCamphuijsen. The changes look pretty nice and the final layout amazing! I tested it locally. One change I cannot really grasp is the `id`. Will `id` be replaced by `order` also in `index.md` ? 

I will approve for now...

Reply:
Thank you for your review @Cmurilochem . Indeed, I replaced the `id` variable with `order` because it is more descriptive of how it is used.